tests/www/test_utils.py

```diff
@@ -178,11 +177,11 @@ def test_markdown(self):
         assert "<li>bar</li>" in rendered
 
     def test_markdown_none(self):
-        rendered = self.attr_renderer["python_callable"](None)
-        assert "" == rendered
+        rendered = self.attr_renderer["doc_md"](None)
+        assert rendered is None
```

Comment:
Also "fix" this test seems like it just copy of `test_python_callable_none` but name of the test more related to `doc_md` attribute